tests/socket.rs

```diff
@@ -101,20 +101,23 @@ fn type_fmt_debug() {
 
 #[test]
 fn protocol_fmt_debug() {
-    let tests = &[
+    let mut tests = vec![
         (Protocol::ICMPV4, "IPPROTO_ICMP"),
         (Protocol::ICMPV6, "IPPROTO_ICMPV6"),
         (Protocol::TCP, "IPPROTO_TCP"),
         (Protocol::UDP, "IPPROTO_UDP"),
         (500.into(), "500"),
     ];
 
+    #[cfg(target_os = "linux")]
+    tests.push((Protocol::MPTCP, "IPPROTO_MPTCP"));
```

Comment:
I don't think you have to make `tests` a vector. I think you can do the following
```rust
        #[cfg(target_os = "linux")]
        (Protocol::MPTCP, "IPPROTO_MPTCP"),
```

Reply:
You're rigth! I've updated the commit :)